docs/features/peering.md

```diff
@@ -42,9 +42,9 @@ If this is not possible, refer to the advanced guide to learn how to perform the
 ```
 
 ```{warning}
-Starting from Liqo 1.0 the concept of *out-of-band* and *in-band* peering is deprecated.
 The user can adopt different peering approaches depending if it has contemporary access to both clusters or not, as described in the [dedicated page](/advanced/peering-strategies.md). 
 
-If the user can not expose the API server to the public, there is the possibility to leverage the Liqo networking to let the API server traffic flow into the cross-cluster VPN tunnel and use the *liqo-gateway* as a proxy for the API server.
-The procedure to achieve this will be explained in detail in the next releases.  
+If the user can not expose the API server to the public, there is the possibility to leverage the Liqo networking to let the API server traffic flow into the cross-cluster VPN tunnel by leveraging the [IP remapping](../advanced/external-ip-remapping.md) (of the `liqo-proxy` service) and the [API Server proxy](../advanced/api-server-proxy.md) features.
```

Comment:
```suggestion
If the user cannot expose the API server to the public, you can leverage the Liqo networking to let the API server traffic flow into the cross-cluster VPN tunnel by leveraging the [IP remapping](../advanced/external-ip-remapping.md) (of the `liqo-proxy` service) and the [API Server proxy](../advanced/api-server-proxy.md) features.
```

Reply:
I'm not sure I would understand how to configure this feature. Would it be possible to provide an example?

Reply:
In a default installation, the only required step is to add a `--in-band` flag to liqoctl peer or liqoctl authenticate commands, as explained in the following sentence.